extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -102,9 +102,10 @@ export function isVisibilitySpecValid(config) {
 
   const spec = config['visibilitySpec'];
   const selector = spec['selector'];
-  if (!selector || (selector[0] != '#' && selector.indexOf('amp-') != 0)) {
-    user().error(TAG_, 'Visibility spec requires an id selector or a tag ' +
-        'name starting with "amp-"');
+  if (!selector || (selector[0] != '#' && selector.indexOf('amp-') != 0 &&
```

Comment:
[11:53:47] Not available because we do not currently ship with a needed ES6 polyfill. Is `string.startsWith` not the right form?